app/views/SettingsView/index.js

```diff
@@ -83,6 +98,17 @@ export default class SettingsView extends React.Component {
 		);
 	}
 
+	renderCrashlyticsSwitch = () => {
```

Comment:
We should do it same way as iOS native does:

- "Send Crash Report"
- A description of it

<img src="https://user-images.githubusercontent.com/804994/61708035-2dc6b000-ad22-11e9-92b5-a1c7025213ad.jpeg" width="350" />

Also, remove "Crashlytics" word from other places like methods, actions, etc.

Reply:
nice!

Reply:
fixed.